osmo-test-5/osmo-test-5.chainlist.json

```diff
@@ -848,6 +848,61 @@
         "ibc-transfer",
         "ibc-go"
       ]
+    },
+    {
```

Comment:
There are inconsistencies between the update to this chainlist file and the zone_chains file.
It's best not to update this chainlist file directly at all.

Reply:
So better I revert all changes on chainlist or fixed the difference?

Reply:
Probably best to: revert all changes on chainlist